### PR TITLE
LinusHomeworkFixed.md

### PR DESCRIPTION
**Pull request by Bahia369**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

**Review by mikinet — CHANGES_REQUESTED** [review]
Hi Bahia, I think this answer about JS in Q.9 is just fine, but you can make it even much better if you mention something special about JS. Cheers.

File: homework.md [hunk 1]
```diff
@@ -1,34 +1,36 @@
...
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
```
Comment (jip1984) [inline homework.md:27]: Hypertext is one word (Just commenting for homework purposes)

File: homework.md [hunk 2]
```diff
@@ -56,25 +58,25 @@ When people want to show off code to each other they can put it on Github
...
 
-It send it to a friend
+you take it from github online to github desktop and is available to work on
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+When you send a file over the internet and is allowed t obe viewed and edited by another person
```
Comment (jip1984) [inline homework.md:73]: You make local code changes and then submit those changes to a remote project maintainer for review before those changes are implemented, or merged. This is called a pull request; you are requesting that someone reviews and approves your changes before they become final.

File: homework.md [hunk 3]
```diff
@@ -56,25 +58,25 @@ When people want to show off code to each other they can put it on Github
...
 
 ## Question 12
 
 What is the Git History log?
```
Comment (samia-pk) [inline homework.md:77]: Git History log: it shows you all the history of changes in a file.

please leave feedback on my work also. thanks

File: homework.md [hunk 4]
```diff
@@ -44,7 +46,7 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
```
Comment (samia-pk) [inline homework.md:47]: JavaScript: it helips you to make your web page more interactive in various way.

please leave feedback on my work also. thanks

File: homework.md [hunk 5]
```diff
@@ -56,25 +58,25 @@ When people want to show off code to each other they can put it on Github
...
+you take it from github online to github desktop and is available to work on
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+When you send a file over the internet and is allowed t obe viewed and edited by another person
 
 ## Question 12
```
Comment (samia-pk) [inline homework.md:75]: Git History log: it shows you all the history of changes in a file.

please leave feedback on my work also. thanks

**Pull request closed without merging**